coremltools/converters/mil/frontend/torch/torch_op_registry.py

```diff
@@ -127,3 +129,18 @@ def func_wrapper(func):
         # decorator called without argument
         return func_wrapper
     return func_wrapper(_func)
+
+
+def is_torch_fx_node_supported(
+    torch_fx_node: torch.fx.Node,
+    skip_ops: List[str] = [],
+) -> bool:
+    assert torch_fx_node.op == "call_function"
+    
+    torch_fx_node_target_name = torch_fx_node.target.__name__.lower()
+    if torch_fx_node_target_name.startswith("aten."):
+        torch_fx_node_target_name = torch_fx_node_target_name[5:]
+    
+    if torch_fx_node_target_name in skip_ops:
+        return False
+    return torch_fx_node_target_name in _TORCH_OPS_REGISTRY
```

Comment:
Is `_TORCH_OPS_REGISTRY` stored somewhere?

Reply:
Yes, in here: https://github.com/apple/coremltools/blob/main/coremltools/converters/mil/frontend/torch/torch_op_registry.py#L97